ote/src/cljs/ote/views/route/route_list.cljs

```diff
@@ -45,91 +51,152 @@
+     [ui/table-header-column {:class "table-header hidden-xs hidden-sm " :style {:width "15%"}} (tr [:route-list-page :route-list-table-created])]
+     [ui/table-header-column {:class "table-header hidden-xs hidden-sm " :style {:width "11%"}} (tr [:route-list-page :route-list-table-actions])]]]
+   [ui/table-body {:display-row-checkbox false}
+    (if empty-row-text
+      (doall
+        [ui/table-row {:key (str empty-row-text) :selectable false :display-border false}
+         [ui/table-row-column {:style {:width "100%"}} empty-row-text]
+         [ui/table-row-column ""]
+         [ui/table-row-column ""]
+         [ui/table-row-column ""]])
+      (doall
+        (map-indexed
+          (fn [i {::transit/keys [id name published? available-from available-to
+                                  departure-point-name destination-point-name]
+                  ::modification/keys [created modified] :as row}]
+            ^{:key (str "route-" i)}
+            [ui/table-row {:key (str "route-" i) :selectable false :display-border false}
+             [ui/table-row-column {:style {:width "18%"}}
+              [:a {:href "#"
+                   :on-click #(do
+                                (.preventDefault %)
+                                (e! (fp/->ChangePage :edit-route {:id id})))} (t-service/localized-text-with-fallback @selected-language name)]]
+             [ui/table-row-column {:class "hidden-xs hidden-sm " :style {:width "15%"}} (time/format-timestamp-for-ui modified)]
+             [ui/table-row-column {:class "hidden-xs hidden-sm " :style {:width "15%"}} (time/format-timestamp-for-ui created)]
+             [ui/table-row-column {:class "hidden-xs hidden-sm " :style {:width "13%"}}
+              [ui/icon-button {:href "#"
+                               :on-click #(do
+                                            (.preventDefault %)
+                                            (e! (fp/->ChangePage :edit-route {:id id})))}
+               [ic/content-create]]
+              [delete-route-action e! row]]])
+          routes)))]])
+
+(defn service-linked-to-route [service services-with-route]
+  (let [my-list (some
+                  (fn [service-with-route]
+                    (= (:id service-with-route)
+                       (::t-service/id service)))
+                  services-with-route)]
+    (if my-list true false)))
+
+(defn- link-services-to-routes [e! services services-with-route has-public-routes]
+  (if has-public-routes
+    [:div
+     [:h4 (tr [:route-list-page :header-link-public-routes-to-service])]
+     [:p (tr [:route-list-page :select-services-to-link-interfaces])]
+
+     (when (empty? services-with-route)
+       [:div
+        [:span (stylefy/use-style style-base/icon-with-text)
+         [ic/alert-warning {:style {:color colors/negative-button
+                                    :margin-right "0.5rem"
+                                    :margin-bottom "5px"}}]
```

Comment:
Different margin units mixed here, both should use `rem`?

Reply:
Changed to 0.25rem